src/services/DavRequest.ts

```diff
@@ -39,13 +40,17 @@ const props = `
 	<nc:nbItems />
 `
 
-export { props }
-export default `<?xml version="1.0"?>
+/**
+ * @param extraProps - Extra properties to add to the DAV request.
+ */
+export function getPropFind(extraProps: string[] = []): string {
+	return `<?xml version="1.0"?>
 			<d:propfind xmlns:d="DAV:"
 				xmlns:oc="http://owncloud.org/ns"
 				xmlns:nc="http://nextcloud.org/ns"
 				xmlns:ocs="http://open-collaboration-services.org/ns">
 				<d:prop>
-					${props}
+					${[...davProps, ...extraProps].join('')}
```

Comment:
Can we use the `@nc/file` default props list?

Reply:
partly yes